packages/commerce-server/src/record-new-purchase.ts

```diff
@@ -29,16 +29,9 @@ export class PurchaseError extends Error {
 }
 
 export async function stripeData(checkoutSessionId: string) {
-  const checkoutSession = await stripe.checkout.sessions.retrieve(
-    checkoutSessionId,
-    {
-      expand: [
-        'customer',
-        'line_items.data.price.product',
-        'payment_intent.charges',
-      ],
-    },
-  )
+  const {getCheckoutSession} = getStripeSdk()
+
+  const checkoutSession = await getCheckoutSession(checkoutSessionId)
```

Comment:
we should refactor to this style for accessing service APIs through 2nd/3rd party libs imo